SparkPost/2web2ui
# FE-965 - Hibana style cleanup for account settings, cancel, profile, and SMTP pages

[FE-965](https://jira.int.messagesystems.com/browse/FE-965)

### What Changed
- Updated components and styles to look AMAZING for Hibana for the following routes:
  - /account/settings
  - /account/cancel - **be careful with this one - I wouldn't use `appteam` to test this - create a new account**
  - /account/profile
  - /account/smtp
  - _AND_ the support modal

### How To Test
- Visit the above routes and verify things look good on both the OG and Hibana versions of the app

### To Do
- [x] Fix out of date unit tests
- [x] Waiting on merging the RV restyle ticket for some updates to the `ExternalLink` component
